examples/step-74/doc/intro.dox

```diff
@@ -0,0 +1,119 @@
+$\average{v}=v$.
+The discretization using the SIPG is given by the following weak formula
+(more details can be found in @cite di2011mathematical and the references therein)
+@f{multline*}
+  \sum_{K\in \Gamma_h} (\nabla v_h, \nu \nabla u_h)_K
+  \\
+  - \sum_{F \in F_h^i} \biggl\{
+    \bigl< \jump{v_h}, \nu\average{ \nabla u_h} \cdot  \mathbf n \bigr>_F
+   +\bigl<\average{ \nabla v_h }\cdot \mathbf n,\nu\jump{u_h}\bigr>_F
+   -\bigl<\jump{v_h},\nu \sigma \jump{u_h} \bigr>_F
+  \biggr\}
+  \\
+  - \sum_{F \in F_h^b} \biggl\{
+    \bigl<v_h, \nu  \nabla u_h\cdot \mathbf n \bigr>_F
+  + \bigl< \nabla v_h \cdot \mathbf n , \nu u_h\bigr>_F
+  - \bigl< v_h,\nu \sigma u_h\bigr>_F
+  \biggr\}
+  \\
+  = (v_h, f)_\Omega
+  - \sum_{F \in F_h^b} \biggl\{
+    \bigl< \nabla v_h \cdot \mathbf n, \nu g_D\bigr>_F - \bigl<v_h,\nu \sigma g_D\bigr>_F
+  \biggr\}.
+@f}
+
+<h3>The penalty parameter</h3>
+The parameter is defined as $\sigma = \gamma/h_f$, where $h_f$ a local length scale associated
+with the cell face, here we choose the length of the cell in the direction normal to the face,
+and $\gamma$ is the penalization constant.
+To ensure the discrete coercivity, the penalization constant has to be large enough @cite ainsworth2007posteriori.
+People do not really have consensus on which precise formula to choose, among what was proposed in the literature.
+One can just pick a large constant, while other options could be the multiples of $(p+1)^2$ or $p(p+1)$. In this code,
+we follow step-39 and use $\gamma = p(p+1)$.
+
+<h3>Posteriori error estimator</h3>
+In this example, we use the error estimator by Karakashian and Pascal @cite karakashian2003posteriori
+@f[
+\eta^2 = \sum_{K \in \Gamma_h} \eta^2_{K} +  \sum_{f_i \in F^i_h}  \eta^2_{f_i} + \sum_{f_b \in F^i_b}\eta^2_{f_b}
+@f]
+where
+@f[
+\eta^2_{K} = h_K^2 \left\| f + \nu \Delta u_h \right\|_K^2
+@f]
+@f[
+\eta^2_{f_i} = \sigma \left\| \jump{u_h}  \right\|_f^2   +  h_f \left\|  \jump{\nu \nabla u_h} \cdot \mathbf n   \right\|_f^2
+@f]
+@f[
+\eta_{f_b}^2 =  \sigma \left\| u_h-g_D \right\|_f^2
+@f]
+The only difference is that we use $\sigma = \gamma/h_f$ instead of $\gamma^2/h_f$ for the jump terms of $u$ (the first term in $\eta^2_{f_i}$ and $\eta_{f_b}^2$).
+
+In each cell $K$, we compute
+@f[
+\eta_{c}^2 = h_K^2 \left\| f + \nu \Delta u_h \right\|_K^2,
+@f]
+@f[
+\eta_{f}^2 = \sum_{f\in \partial K}\lbrace \sigma \left\| \jump{u_h}  \right\|_f^2   +  h_f \left\|  \jump{\nu \nabla u_h} \cdot \mathbf n  \right\|_f^2 \rbrace,
+@f]
+@f[
+\eta_{b}^2 = \sum_{f\in \partial K \cap \partial \Omega}  \sigma \left\| (u_h -g_D)  \right\|_f^2.
+@f]
+Then the error estimate square per cell is
+@f[
+\eta_{local}^2 =\eta_{c}^2+0.5\eta_{f}^2+\eta_{b}^2.
+@f]
+Note that we compute $\eta_{local}^2$ instead of $\eta_{local}$ to simplify the implementation.
+The error estimate square per cell is store in a global vector, whose $L_1$ norm is equal to $\eta^2$.
```

Comment:
```suggestion
The error estimate square per cell is stored in a global vector, whose $L_1$ norm is equal to $\eta^2$.
```

Reply:
I think it would be useful to extend the introduction a bit to explain what the error estimator is used for,  the testcase, etc. Right now, it ends a bit abruptly.